Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -265,7 +268,19 @@ namespace AzToolsFramework
             auto value = AZ::DocumentPropertyEditor::Nodes::PropertyEditor::Value.ExtractFromDomNode(node);
             if (value.has_value())
             {
-                m_proxyValue = AZ::Dom::Utils::ValueToType<WrappedType>(value.value()).value_or(m_proxyValue);
+                if (!AzToolsFramework::Prefab::IsInspectorOverrideManagementEnabled())
```

Comment:
this layer is just for RPE-specific stuff. This is very much a bridge to the old system. Please find an Inspector-specific place to put this so it doesn't bleed into generic PropertyEditor stuff

Reply:
This looks like a gradual switch-over to loading JSON-serialized values here which is very cool! Maybe we can just start doing that everywhere instead of gating it on overrides? There aren't that many other DPE adapters so I'm expected a light lift, but I also could be super wrong - maybe there are some reflected types that will really struggle with this - long term I'd love to see these all just marshal serialized values in and out though

Reply:
Yes, this will eventually be the new way of loading opaque values from DPE DOM. This isn't inspector only logic. The check is only temporary and is put in place so that we can do enough testing and gain confidence instead of changing everything at once.